.golangci.yml

```diff
@@ -11,27 +9,44 @@ linters:
     - errcheck
     - gocritic
     - gocyclo
-    - gofmt
-    - goimports
-    - gosec
     - goprintffuncname
-    - gosimple
+    - gosec
     - govet
     - ineffassign
     - misspell
     - nakedret
     - nolintlint
     - revive
     - staticcheck
-    - stylecheck
-    - typecheck
```

Comment:
`typecheck` is essentially removed as it has been merged to `Standard Linters` which is disabled by `default: none`.